datawire-envoy-1.31.yaml

```diff
@@ -69,6 +69,15 @@ pipeline:
 
   - uses: strip
 
+subpackages:
+  - name: ${{package.name}}-setcap-ei
+    description: Envoy binary with CAP_NET_BIND_SERVICE capabilities enabled, required for downstream Ambassador usage.
+    pipeline:
+      - runs: |
+          mkdir -p "${{targets.subpkgdir}}"/usr/bin
+          cp -p "${{targets.destdir}}"/usr/bin/envoy "${{targets.subpkgdir}}"/usr/bin/envoy
+          setcap cap_net_bind_service=ei /usr/local/bin/envoy
```

Comment:
Should both of these be /usr/bin/envoy or /usr/local/bin/envoy?

Reply:
Good catch. The upstream dockerfile uses /usr/local/bin, but IIUC we prefer /usr/bin because we consider these distribution provided?

Changed to /usr/bin.